Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/MenuViewController.swift

```diff
@@ -201,9 +203,16 @@ class MenuViewController: UITableViewController {
     }
     
     private func openAddBookmark() {
-        guard let title = tab?.displayTitle, let url = tab?.url?.absoluteString else { return }
+        guard let title = tab?.displayTitle, let url = tab?.url else { return }
+        
+        let mode: BookmarkEditMode = {
```

Comment:
Extract only `url` from this. `BookmarkEditMode.addBookmark(title: title` always stays the same for both conditions so we only need to set a correct url string

can be done as simply as 
```
var bookmarkUrl = url.absoluteString
if url.isReaderModeURL, let decodedURL = url.decodeReaderModeURL { bookmarkUrl = decodedURL.absoluteString }
```